lib/tasks/last_interaction.rake

```diff
@@ -0,0 +1,14 @@
+namespace :last_interaction do
+  desc "seeds more appropriate last_interaction value onto all existing clients."
+  task "backfill" => :environment do
+    # break job into separate processable jobs in batches of 1000.
+    (0..Client.all.size).step(1000) do |i|
```

Comment:
I knew about `.count` vs `.length` but this is the first I learned of `.size`! interesting!